tezos_interop/tezos_interop.re

```diff
@@ -422,6 +489,59 @@ module Consensus = {
       address(ticketer),
     )
     |> hash_packed_data;
+
+  // TODO: how to test this?
```

Comment:
re: testing

We could start maintaining a mock tezos node for unit testing.

Reply:
(feel free to resolve)

Reply:
#101 good idea